beetsplug/web/__init__.py

```diff
@@ -337,6 +338,11 @@ def func(lib, opts, args):
                     r"/*": {"origins": self.config['cors'].get(str)}
                 }
                 CORS(app)
+
+            if self.config['subsonic']:
+                from .subsonic import subsonic_routes
+                app.register_blueprint(subsonic_routes)
+
```

Comment:
Yes, this seems like it could work! The more typical strategy for beets would be to define a method on the plugin class to get a set of blueprints, which could also work here—the blueprint would still be defined at the module level, but it could be registered in a callback. That's worth exploring!